src/System.Drawing.Common/tests/StringFormatTests.cs

```diff
@@ -0,0 +1,422 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing.Text;
+using Xunit;
+
+namespace System.Drawing.Tests
+{
+    public class StringFormatTests
+    {
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Ctor_Default()
+        {
+            var format = new StringFormat();
+            Assert.Equal(StringAlignment.Near, format.Alignment);
+            Assert.Equal(0, format.DigitSubstitutionLanguage);
+            Assert.Equal(StringDigitSubstitute.User, format.DigitSubstitutionMethod);
+            Assert.Equal((StringFormatFlags)0, format.FormatFlags);
+            Assert.Equal(HotkeyPrefix.None, format.HotkeyPrefix);
+            Assert.Equal(StringAlignment.Near, format.LineAlignment);
+            Assert.Equal(StringTrimming.Character, format.Trimming);
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [InlineData(StringFormatFlags.DirectionRightToLeft | StringFormatFlags.DirectionVertical)]
+        [InlineData((StringFormatFlags)(-1))]
+        public void Ctor_Options(StringFormatFlags options)
+        {
+            var format = new StringFormat(options);
+            Assert.Equal(StringAlignment.Near, format.Alignment);
+            Assert.Equal(0, format.DigitSubstitutionLanguage);
+            Assert.Equal(StringDigitSubstitute.User, format.DigitSubstitutionMethod);
+            Assert.Equal(options, format.FormatFlags);
+            Assert.Equal(HotkeyPrefix.None, format.HotkeyPrefix);
+            Assert.Equal(StringAlignment.Near, format.LineAlignment);
+            Assert.Equal(StringTrimming.Character, format.Trimming);
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [InlineData(StringFormatFlags.DirectionRightToLeft | StringFormatFlags.DirectionVertical, 10)]
+        [InlineData(StringFormatFlags.NoClip, 2057)]
```

Comment:
Can you make these language values const variables so it's clear what language they represent? It's not critical to the logic of the test, but it would make things less opaque.